applications/matter_bridge/README.rst

```diff
@@ -36,9 +36,20 @@ The application supports bridging the following Matter device types:
 * Temperature Sensor
 * Humidity Sensor
 
-The application simulates the functionality of bridged devices by generating pseudorandom measurement results and changing the light state in a fixed time intervals.
+The application supports two bridged devices configurations that are mutually exclusive and can be enabled using the following Kconfig options:
+
+* :kconfig:option:`CONFIG_BRIDGED_DEVICE_SIMULATED` - simulates the functionality of bridged devices by generating pseudorandom measurement results and changing the light state in a fixed time intervals.
+* :kconfig:option:`CONFIG_BRIDGED_DEVICE_BLE` - allows to connect a real peripheral Bluetooth LE devices to the bridge and represent their functionalities using Matter Data Model.
+
 The bridge supports adding and removing bridged devices dynamically at application runtime using commands from a dedicated Matter bridge shell module.
 
+Bluetooth LE Bridged devices
+============================
+
+The application supports following Bluetooth LE services that are represented by the corresponding Matter device types:
+
+* :ref:`Nordic Led Button Service <lbs_readme>` represented by the Matter On/Off Light
+
```

Comment:
I would rather have all of this as one section "Bridged devices", in which I would explain what the bridged devices are and what types are implemented by the sample. Then, I would move the Kconfigs to a subsection of Configuration "Enabling bridged devices".